src/lib/netlist/FAQ.md

```diff
@@ -0,0 +1,150 @@
+# The netlist FAQ
+
+## Glosary
+
+BTANB: Bugs That Aren't Bugs.
+
+## General
+
+### Have you considered maybe supporting part randomization in the netlist core?
+
+Yes I did. And decided against it. Which features? Forced randomness - will this
+be different at every restart? And it would require another parameter per device
+(specification, e.g. for resistors 5%, 10%, 20%). This would cause more cache 
+usage.
+
+And I am convinced we would see more BTANBs.
+
+### How do frontiers work?
+
+Frontiers divides a netlist into sub netlists, i.e. a number of smaller netlists.
+Frontiers work best if you have a low impedance to high impedance transition. 
+This is best illustrated by an example. Consider the following mixing stage
+	
+	                 R1
+	      S1 >-----1RRRR2---------+
+	                              |
+	                 R2           |
+	      S2 >-----1RRRR2---------+----------> Out
+	                              |
+	                              R
+	                           R3 R
+	                              R
+	                              |
+	                             GND
+	
+With `OPTIMIZE_FRONTIER(R2.2, R3, R2)` this becomes:
+	
+	                 R1
+	      S1 >-----1RRRR2--------------------------------+
+	                                                     |
+	                       ##########################    |
+	                 R2    #                    R2  #    |
+	      S2 >-----1RRRR2-----+-->AnIn AnOut>--RRRR------+----------> Out
+	                       #  |                     #    |
+	                       #  R                     #    R
+	                       #  R R3                  # R3 R
+	                       #  R                     #    R
+	                       #  |                     #    |
+	                       # GND          Frontier  #   GND
+	                       #                        #
+	                       ##########################
+	
+As a result, provided there are no other connections between the parts
+generating S1 and S2 the "S2 part" will now have a separate solver.
+
+The size (aka number of nets) of the solver for S1 will be smaller.
+The size of the solver for S2 and the rest of the circuit will be smaller
+as well.
+
+Frontiers assume that there is feedback from the "out" terminal to the "in" 
```

Comment:
Assume that there is feedback, or no feedback?

Reply:
little to no feedback. Fixed